### PR TITLE
Use real URLs for links in lists

### PR DESCRIPTION
**Pull request by alex-deaconu**
[stage-4]

### CODE REVIEW

**Comment by alex-deaconu** [comment]
This is a pretty straight forward PR. The major change is in the CH version: https://github.com/Rise-Vision/common-header/pull/608.

To summarize:
- Adding the `?cid` parameter to the url paths for schedules/displays/editor allows it to be added to URLs generated via the `ui-sref` directives
- Updated list pages and Add buttons to use a hyperlink. This allows users to shift-click (cmd-click) to open in a new tab/window.
- Removed `ui-router` dependency as it is loaded by the CH now

@ezequielc please review. @olegrise FYI. Thanks.

File: web/partials/schedules/schedules-list.html [hunk 1]
```diff
@@ -37,17 +37,17 @@ <h1 id="title" translate>schedules-app.title</h1>
         </tr>
       </thead>
       <tbody>
-        <tr class="clickable-row" ng-repeat="schedule in schedules.list" ui-sref="apps.schedules.details({scheduleId: schedule.id})">
-          <td class="col-sm-9"><strong>{{schedule.name}}</strong></td>
+        <tr class="clickable-row" ng-repeat="schedule in schedules.list">
+          <td class="col-sm-9"><a ui-sref="apps.schedules.details({scheduleId: schedule.id})"><strong>{{schedule.name}}</strong></a></td>
           <td class="col-sm-2 hidden-xs text-right nowrap">{{schedule.changeDate | date:'d-MMM-yyyy h:mm a'}}</td>
```
Comment (ezequielc) [inline web/partials/schedules/schedules-list.html]: @alex-deaconu I think it is missing the link here. Cheers

Comment (alex-deaconu) [inline web/partials/schedules/schedules-list.html]: @ezequielc good catch. Fixed. Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu Other than that comment, everything looks good! :+1: Cheers!

**Pull request merged**